### ISSUE TITLE
Text summarization for article content

### ISSUE THREAD

**Issue by arvindpdmn** [DevopediaOrg/webapp#469]
- Given text from references, generate a summary.
- "Summary"" is meant for Summary and Discussion sections. For Discussion, summaries have to be "clustered" and brought under suitable questions. It may be related to #428

**Comment by arvindpdmn** [comment]
Some useful resources:
- https://devopedia.org/text-summarization
- https://devopedia.org/question-answering
- Survey papers:
  - https://arxiv.org/abs/2011.04843
  - https://american-cse.org/sites/csci2020proc/pdfs/CSCI2020-6SccvdzjqC7bKupZxFmCoA/762400b503/762400b503.pdf
- http://pfliu.com/Historiography/summarization/summ-eng.html
- http://nlpprogress.com/english/summarization.html
- Routing Transformer and REALM: https://ai.googleblog.com/2021/03/progress-and-challenges-in-long-form.html
- PEGASUS: https://ai.googleblog.com/2020/06/pegasus-state-of-art-model-for.html
- T5: https://ai.googleblog.com/2020/02/exploring-transfer-learning-with-t5.html
- CAiRE-COVID: https://arxiv.org/abs/2005.03975
- BART: https://arxiv.org/pdf/1910.13461.pdf
 - TLDR: https://arxiv.org/abs/2004.15011
- Graph models:
  - https://arxiv.org/abs/1811.01824
  - https://www.aclweb.org/anthology/P04-3020.pdf
- Generating Wikipedia: https://arxiv.org/abs/1801.10198
- WikiAsp: Aspect-based summarization: https://direct.mit.edu/tacl/article-pdf/doi/10.1162/tacl_a_00362/1894310/tacl_a_00362.pdf
- General reading: 
  - https://cacm.acm.org/magazines/2021/4/251336-the-best-of-nlp/fulltext
  - https://web.ntpu.edu.tw/~myday/teaching/1102/AITA/1102AITA07_AI_for_Text_Analytics.pdf
